apps/web/app/(booking-page-wrapper)/team/[slug]/[type]/actions.ts

```diff
@@ -0,0 +1,53 @@
+"use server";
+
+import { revalidatePath } from "next/cache";
+import { unstable_cache } from "next/cache";
+
+import { NEXTJS_CACHE_TTL } from "@calcom/lib/constants";
+import { EventTypeService } from "@calcom/lib/server/service/eventType";
+import { TeamService } from "@calcom/lib/server/service/team";
+import type { TeamWithEventTypes } from "@calcom/lib/server/service/team";
+
+export async function revalidateTeamBookingPage(
```

Comment:
we call this cache invalidation when an event-type is updated (e.g., in `EventTypeWebWrapper.tsx`)